README.md

```diff
@@ -82,7 +82,7 @@ For a complete documentation go to [Realm Flutter and Dart SDK Docs](https://doc
 
 ## Limitations
 
-* This version of Realm Flutter and Dart SDK allows working with a local only (on device) Realm database in Flutter and Dart desktop. Realm Sync functionality is not implemented.
+* This version of Realm Flutter and Dart SDK allows working with a local only (on device) Realm database in Flutter and Dart desktop. Atlas Device Sync functionality is not implemented.
```

Comment:
Should we still list that limitation? It should be gone with the next release, so might as well just delete it from the readme.

Reply:
No we should have this limitation any more. Will fix